go-controller/pkg/libovsdbops/model.go

```diff
@@ -227,6 +235,16 @@ func copyIndexes(model model.Model) model.Model {
 		return &sbdb.IGMPGroup{
 			UUID: t.UUID,
 		}
+	case *sbdb.Encap:
+		return &sbdb.Encap{
+			UUID:        t.UUID,
+			ChassisName: t.ChassisName,
```

Comment:
ChassisName should not be listed here because it is not an index. IP and Type should be listed.